public/dist/style.css

```diff
@@ -2,14 +2,21 @@ div {
   display: block;
 }
 
+.checkDates {
+  cursor: pointer;
+}
 .widget {
   position: relative;
   width: 319px;
   font-family: "Calibre", Helvetica, Arial, sans-serif;
   /* look into what these are doing: */
   z-index: 99;
```

Comment:
setting a greater z-index puts that particular styled component in front of everything else; that was key to getting my modal behaving as I expected